app/views/providers/legal_aid_applications/_proceeding_type.html.erb

```diff
@@ -0,0 +1,24 @@
+<div id="<%= proceeding_type.code %>" class="govuk-grid-row proceeding-item" style="display: block;">
```

Comment:
**[not a blocker/minor]** This was probably in the original JS template, but if we can avoid the use of embedded styling (`style="display: block;"`) in the page it would be 🏅 .

Reply:
I kept this bit of markup as it was, but I've just tested and in fact the style is defaulting to `block` anyway. So I've removed it.